pkg/logging/logging.go

```diff
@@ -92,3 +93,20 @@ func (l logrLogger) Debug(msg string, keysAndValues ...any) {
 func (l logrLogger) WithValues(keysAndValues ...any) Logger {
 	return logrLogger{log: l.log.WithValues(keysAndValues...)} //nolint:logrlint // False positive - logrlint thinks there's an odd number of args.
 }
+
+// ForResource returns logging values for a resource.
+func ForResource(object client.Object) []string {
```

Comment:
it seems that this is not used anywhere in the code?

Reply:
but should be, see you comment above about the error context.

Reply:
added